src/wazuh_db/wdb_parser.c

```diff
@@ -5369,7 +5382,7 @@ int wdb_parse_global_sync_agent_groups_get(wdb_t* wdb, char* input, char* output
             } else if (0 == strcmp(j_sync_condition->valuestring, "all")) {
                 condition = WDB_GROUP_ALL;
             }
-            if (cJSON_IsNumber(j_last_id)){
+            if (j_last_id && cJSON_IsNumber(j_last_id)){
```

Comment:
Although this check shouldn't be necessary, it was included to avoid having `scan-build` issues.

Reply:
We can remove the `cJSON_IsNumber()`check, at this point if the value isn't NULL, we know it has the right type of data